yb-voyager/cmd/importSchema.go

```diff
@@ -192,7 +209,46 @@ func importSchema() {
 	importSchemaCompleteEvent := createImportSchemaCompletedEvent()
 	controlPlane.ImportSchemaCompleted(&importSchemaCompleteEvent)
 
-	callhome.PackAndSendPayload(exportDir)
+	return nil
+}
+
+func packAndSendImportSchemaPayload(status string, errMsg string) {
+	if !callhome.SendDiagnostics {
+		return
+	}
+	payload := createCallhomePayload()
+	payload.MigrationPhase = IMPORT_SCHEMA_PHASE
+	payload.Status = status
+	targetDBDetailsBytes, err := json.Marshal(targetDBDetails)
+	if err != nil {
+		log.Errorf("callhome: error in parsing sourcedb details: %v", err)
+	}
+	payload.TargetDBDetails = string(targetDBDetailsBytes)
+	var errorsList []string
+	for _, stmt := range finalFailedSqlStmts {
```

Comment:
consider adding a comment explaining this for-loop(split and trim parts) with an example.